staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_internal.go

```diff
@@ -225,6 +220,16 @@ func (g *Cloud) ensureInternalLoadBalancer(clusterName, clusterID string, svc *v
 		}
 	}
 
+	fwd, err := g.GetRegionForwardingRule(loadBalancerName, g.region)
```

Comment:
Pasting comment from offline - merge the second `GetRegionForwardingRule` down below into this so we don't generate additional get call.